FAILED test.py::test_set_get_policy_aes_256_xts - Failed: DID NOT RAISE <class 'OSError'>

We starting to get such error on our tests:

```
root@i586:~/RPM/BUILD/fscryptctl-1.0.0# make test-all
make test-setup
make[1]: Entering directory '/usr/src/RPM/BUILD/fscryptctl-1.0.0'
if mountpoint -q "/tmp/fscryptctl-test-dir"; then \
        sudo umount "/tmp/fscryptctl-test-dir"; \
fi
0.00user 0.00system 0:00.01elapsed 10%CPU (0avgtext+0avgdata 2060maxresident)k
0inputs+8outputs (0major+107minor)pagefaults 0swaps
rm -rf "/tmp/fscryptctl-test-dir"
rm -f "/tmp/fscryptctl-test-image"
dd if=/dev/zero of="/tmp/fscryptctl-test-image" bs=1M count=32
32+0 records in
32+0 records out
33554432 bytes (34 MB, 32 MiB) copied, 0.00999589 s, 3.4 GB/s
mkfs.ext4 -b 4096 -O encrypt -F "/tmp/fscryptctl-test-image"
mke2fs 1.46.4 (18-Aug-2021)
Discarding device blocks: done
Creating filesystem with 8192 4k blocks and 8192 inodes

Allocating group tables: done
Writing inode tables: done
Creating journal (1024 blocks): done
Writing superblocks and filesystem accounting information: done

mkdir -p "/tmp/fscryptctl-test-dir"
sudo mount -o rw,loop "/tmp/fscryptctl-test-image" "/tmp/fscryptctl-test-dir"
0.00user 0.00system 0:00.00elapsed 0%CPU (0avgtext+0avgdata 2400maxresident)k
706inputs+8outputs (0major+186minor)pagefaults 0swaps
sudo sh -c 'chown $SUDO_UID:$SUDO_GID "/tmp/fscryptctl-test-dir"'
0.00user 0.00system 0:00.00elapsed 50%CPU (0avgtext+0avgdata 2328maxresident)k
0inputs+0outputs (0major+155minor)pagefaults 0swaps

/tmp/fscryptctl-test-dir is now set up.
make[1]: Leaving directory '/usr/src/RPM/BUILD/fscryptctl-1.0.0'
make test
make[1]: Entering directory '/usr/src/RPM/BUILD/fscryptctl-1.0.0'
TEST_DIR="/tmp/fscryptctl-test-dir" PATH="$PWD:$PATH" \
         ENABLE_VALGRIND="" \
         python3 -m pytest test.py -s -q
............F......................
========================================================================== FAILURES ===========================================================================
_______________________________________________________________ test_set_get_policy_aes_256_xts _______________________________________________________________

directory = '/tmp/fscryptctl-test-dir/test'

    def test_set_get_policy_aes_256_xts(directory):
        """Tests getting and setting an encryption policy that uses AES-256-XTS
        contents encryption and AES-256-CTS filenames encryption.  (Note that this
        is also the default setting, but this test tries it explicitly.)"""
        prepare_encrypted_dir(directory, "--contents=AES-256-XTS",
                              "--filenames=AES-256-CTS")
        check_policy(directory, contents="AES-256-XTS", filenames="AES-256-CTS")
        # AES-256-XTS expects a 64-byte key.  Shorter keys shouldn't work.
        for key in [TEST_KEY_16B, TEST_KEY_32B]:
            with pytest.raises(OSError):
>               prepare_encrypted_dir(directory, "--contents=AES-256-XTS",
                                      "--filenames=AES-256-CTS", key=key)
E               Failed: DID NOT RAISE <class 'OSError'>

test.py:319: Failed
=================================================================== short test summary info ===================================================================
FAILED test.py::test_set_get_policy_aes_256_xts - Failed: DID NOT RAISE <class 'OSError'>
1 failed, 34 passed in 0.49s
make[1]: *** [Makefile:93: test] Error 1
make[1]: Leaving directory '/usr/src/RPM/BUILD/fscryptctl-1.0.0'
make: *** [Makefile:119: test-all] Error 2
```

--- a/test.py
+++ b/test.py
@@ -313,8 +313,9 @@ def test_set_get_policy_aes_256_xts(directory):
     prepare_encrypted_dir(directory, "--contents=AES-256-XTS",
                           "--filenames=AES-256-CTS")
     check_policy(directory, contents="AES-256-XTS", filenames="AES-256-CTS")
-    # AES-256-XTS expects a 64-byte key.  Shorter keys shouldn't work.
-    for key in [TEST_KEY_16B, TEST_KEY_32B]:
+    # AES-256-XTS is only allowed with master keys that are 32 bytes or longer.
+    # Shorter keys shouldn't work.
+    for key in [TEST_KEY_16B]:
         with pytest.raises(OSError):
             prepare_encrypted_dir(directory, "--contents=AES-256-XTS",
                                   "--filenames=AES-256-CTS", key=key)

